reports/Florida.md

```diff
@@ -37,3 +38,20 @@ A police officer grabs a protestor from behind (light grey hoodie), pull him out
 **Links**
 
 * https://twitter.com/ewcfilms/status/1268003421786906627
+
+### Police fire on protesters outside city hall | Date unknown
+
+Police open fire on protesters outside of city hall with teargas, flashbands, and apparently rubber bullets.
+
+**Links**
+
+* https://twitter.com/greg_doucette/status/1269017349727928320
+
+
+### Police box in protesters before curfew for mass arrest | June 3rd
+
+Police form barricades using bicycles to trap protesters leaving before curfew. 
+
+**Links**
+
+* https://twitter.com/filfranca/status/1268343603832774659
```

Comment:
I don't think this video is evidence of police brutality, we can't really see where the protestors are being directed, the video cuts out too soon.